src/Microsoft.Azure.WebJobs.ServiceBus/Config/ServiceBusExtensionConfig.cs

```diff
@@ -54,6 +57,20 @@ public void Initialize(ExtensionConfigContext context)
                 throw new ArgumentNullException("context");
             }
 
+            // Register an exception handler for background exceptions
+            // coming from MessageReceivers.
+            Config.ExceptionHandler = (e) =>
+            {
+                var sbex = e.Exception as ServiceBusException;
```

Comment:
nit: you should be able to inline this (e.g. `(if e.Exception is ServiceBusException sbx && !sbex.IsTransient)`)

Reply:
But then I have to cast the exception twice